src/test/java/com/rakovets/course/java/core/practice/oop/principles/CatTest.java

```diff
@@ -0,0 +1,77 @@
+package com.rakovets.course.java.core.practice.oop.principles;
+
+import com.rakovets.course.java.core.practice.oop.principles.cat.Cat;
+import com.rakovets.course.java.core.practice.oop.principles.cat.Person;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+class CatTest {
+
+    @Test
+    void mew() {
+        Cat tom = new Cat("Tom");
+        String expectedMew = "Mew";
+        String actualMew = tom.mew();
+        Assertions.assertEquals(expectedMew, actualMew);
```

Comment:
You could separate:
* GIVEN
* WHEN
* THEN

Reply:
Why do we need extra spaces?